File: Plugin/src/BuildConfiguration/TypeSource.cs
using System;

namespace BuildConfiguration
{
	/// <summary>
	/// Summary description for TypeSource.
	/// </summary>
	public class TypeSource
	{
		public TypeSource()
		{

		}

		public TypeSource(string name, string fileName)
		{
			_fileName = fileName;
			_name = name;
		}

		private string _fileName ="";

		public string FileName
		{
			get
			{
				return _fileName;
			}
			set
			{
				_fileName = value;
			}
		}

		private string _name ="";

		public string Name
		{
			get
			{
				return _name;
			}
			set
			{
				_name = value;
			}
		}
	}
}

File: Core/Plugin/Eclipse/src/ComposestarEclipsePlugin/Core/BuildConfiguration/TypeSource.java
package ComposestarEclipsePlugin.Core.BuildConfiguration;

/**
 * Class representing a type source. Used by BuildConfigurationManager to build
 * configuration-file.
 */
@Deprecated
public class TypeSource
{

	private String name = "";

	private String filename = "";

	public TypeSource(String Name, String fileName)
	{
		this.name = Name;
		this.filename = fileName;
	}

	public String getName()
	{
		return name;
	}

	public String getFileName()
	{
		return filename;
	}
}
